tests/x11/gb18030.pm

```diff
@@ -65,7 +65,7 @@ sub run {
     # the following fonts preparing steps are documented at:
     # https://confluence.suse.com/display/~kailiu/How+to+prepare+the+system+for+GB18030-2005+certification
     become_root;
-    zypper_call("rm arphic-ukai-fonts arphic-uming-fonts baekmuk-*-fonts noto-*-tc* noto-*-jp* noto-*-kr* wqy* adobe-sourcecodepro-fonts xorg-x11-fonts-converted", exitcode => [0, 104]);
+    zypper_call("rm arphic-ukai-fonts arphic-uming-fonts baekmuk-*-fonts noto-*-tc* noto-*-jp* noto-*-kr* wqy* adobe-sourcecodepro-fonts xorg-x11-fonts-converted xscreensaver", exitcode => [0, 104]);
```

Comment:
Why do we need to remove xscreensaver? To keep test case more robust?

Reply:
Yes, xscreensaver installs fonts that affects characters in "user defined region 1".

Reply:
Sorry, I am not quite clear about this. 
Does this mean if a user wants to use the characters in "user defined region 1" smoothly, the xscreensaver must be removed from the system?

Reply:
Thanks for your explanation. 
LGTM now.